apps/aecore/src/aec_conductor.erl

```diff
@@ -116,6 +121,17 @@ start_link(Options) ->
 stop() ->
     gen_server:stop(?SERVER).
 
+%%%===================================================================
+%%% Test API
+
+-ifdef(TEST).
```

Comment:
Organisational question, except for the use of `?SERVER` this code could just as well go into the test suite? Not sure if it is better, but worth considering.

Reply:
the problem is that would move in the test suite assumption on internal state of the process - especially if records involved. We may actually expose this function also when compiled in non-test mode - let me give further thought to this.

Reply:
You could do a gen_server:call from the test suite instead if you don't want to expose the api.

Reply:
I am addressing this now.